tools/run_experiment.sh

```diff
@@ -75,6 +75,37 @@ if [[ "$EXPERIMENT_MODE" == "full" ]] || [[ "$EXPERIMENT_MODE" == "search" ]]; t
 fi
 
 
+if [[ "$EXPERIMENT_MODE" == "parallel" ]]; then
+  echo "EXPERIMENT_MODE=$EXPERIMENT_MODE"
+
+  docker pull qdrant/vector-db-benchmark:latest
+
+  echo "Starting ci-benchmark-upload container"
+  docker run \
+    --rm \
+    --name ci-benchmark-upload \
+    -v "$HOME/results:/code/results" \
+    qdrant/vector-db-benchmark:latest \
+    python run.py --engines "${ENGINE_NAME}" --datasets "${DATASETS}" --host "${PRIVATE_IP_OF_THE_SERVER}" --no-skip-if-exists --skip-search --skip-configure &
+  UPLOAD_PID=$!
```

Comment:
Why do you say that it will generate two search files? We clearly have `--skip-search` here.

Reply:
The whole experiment generates 2 search results files. This particular line doesn't.